app/views/vacancies/_vacancy.html.haml

```diff
@@ -9,7 +9,10 @@
     %dd.double
       = format_date(vacancy.publish_on)
     %dt= t('jobs.expires_on')
-    %dd= format_date(vacancy.expires_on)
+    - if vacancy.expiry_time.nil?
+      %dd= format_date(vacancy.expires_on)
+    - else 
+      %dd= format_date(vacancy.expires_on) + " at " + vacancy.expiry_time.strftime("%H:%M%P")
```

Comment:
To get the desired format e.g `9:10` we need to use `'%-l':'%-M%P`

Reply:
Good spot, will correct that 👍

Reply:
fixed in eea1018